src/applications/disability-benefits/2346/Components/ConfirmAddress.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getVeteranInformationData } from '../actions';
+
+class ConfirmAddress extends Component {
+  static propTypes = {
+    addressLine1: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    state: PropTypes.string.isRequired,
+    country: PropTypes.string.isRequired,
+    zip: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    getVeteranInformationData: PropTypes.func.isRequired,
+  };
+  static defaultProps = {
+    addressLine1: '',
+    city: '',
+    state: '',
+    country: '',
+    zip: '',
+    email: '',
+    getVeteranInformationData: () => {},
+  };
+  componentDidMount() {
+    this.props.getVeteranInformationData();
+  }
+
+  render() {
+    const { addressLine1, city, state, zip, email } = this.props;
+
+    return (
+      <div>
+        <h1>Confirm your address</h1>
+        <div>
+          <label htmlFor="address1">Address Line 1</label>
+          <input id="address1" type="text" defaultValue={addressLine1} />
+          <label htmlFor="city">City</label>
+          <input type="text" id="city" defaultValue={city} />
+          <label htmlFor="state">State</label>
+          <input type="text" id="state" defaultValue={state} />
+          <label htmlFor="zip">Postal Code</label>
+          <input type="text" id="zip" defaultValue={zip} />
+          <p className="vads-u-margin-top--4">Confirm your email address</p>
+          <input type="email" defaultValue={email} />
+        </div>
```

Comment:
Should _all_ of these inputs be required? I know these will be prefilled, but what if one piece of information is missing, like they accidentally deleted address line 1 before hitting continue?

Reply:
I added that the fields be required in the form.cofig

Reply:
@Mottie We will be using the data here to ship the product(s) to the correct location so it is necessary to have these fields required. The exception will be the address line 2 since not everyone has one.